### PR TITLE
From the level details dialog, open help and tips links in new tab

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Resolves [PLAT-928](https://codedotorg.atlassian.net/browse/PLAT-928) (at least for now).

Currently the links in the help and tip tab open in a dialog over the screen. This is done using an iframe loading the content from curriculum builder. While the ideal behavior here is to open these links in the level details dialog itself, I don't want to deal with an iframe in this dialog, especially as we're going to be working on concept maps in the near future. We should look into a better solution when we do that work.

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/instructions/ResourceLink.jsx [hunk 1]
```diff
@@ -22,8 +23,12 @@ class ResourceLink extends React.Component {
       // Don't open modal, just open link in new tab/window
       return;
     }
-    // Don't open link, just open modal.
     e.preventDefault();
+    if (!!this.props.openReferenceInNewTab) {
```
Comment (dmcavoy) [inline apps/src/templates/instructions/ResourceLink.jsx:27]: Add a test?

File: apps/src/templates/VideoThumbnail.jsx [hunk 2]
```diff
@@ -6,30 +6,35 @@ import {videoDataShape} from './types';
 export default class VideoThumbnail extends Component {
   static propTypes = {
     video: videoDataShape,
-    onClick: PropTypes.func
+    onClick: PropTypes.func,
+    openInNewTab: PropTypes.bool
+  };
+
+  onThumbnailClick = () => {
```
Comment (dmcavoy) [inline apps/src/templates/VideoThumbnail.jsx:13]: add a test?

**Comment by dmcavoy** [comment]
This looks good. I think adding a couple of tests might be worthwhile. Also, is the follow-up work to come back to this with concept maps tracked in Jira or with the PM spec?

File: apps/src/templates/VideoThumbnail.jsx (hunk 2 above)
Comment (bethanyaconnor) [reply apps/src/templates/VideoThumbnail.jsx:13]: ha, turns out neither of these files ever had any tests 🙃 I'll add a couple of basic ones for this file and `ResourceLink`

**Comment by bethanyaconnor** [comment]
@dmcavoy added tests in 3588abfeab2

**Review by dmcavoy — APPROVED** [review]
Code looks good! 

Just one more follow-up to make sure we are tracking this somewhere for the concept maps specing. @tess323

**Comment by bethanyaconnor** [comment]
Created https://codedotorg.atlassian.net/browse/PLAT-1090 to track follow-up work to improve this overlay when we work on concept maps.

**Pull request merged**